ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml

```diff
@@ -51,14 +51,14 @@ ModalPopup {
         }
         contentItem: Item {
             anchors.fill: parent
-            SVGImage {
+            RoundedImage {
                 id: dollarEmoji
                 width: 32
                 height: 32
                 anchors.left: parent.left
                 anchors.leftMargin: 2
                 anchors.verticalCenter: parent.verticalCenter
-                source: Qt.resolvedUrl("../../../../imports/twemoji/svg/1f911.svg")
+                source: appMain.getProfileImage(profileModel.profile.pubKey)
```

Comment:
What does it show if I didn't set a profile image?

Reply:
Nothing. You need to add the identicon if there is no image.
However, since it's the current user's image, you can just use the profile model. It has the image or defaults to the identicon directly.
So it can only be :
```suggestion
                source: profileModel.profile.thumbnailImage
```

This is the model in case you're curious:
```nim
 proc thumbnailImage*(self: ProfileInfoView): string {.slot.} =
    if (?.self.identityImage.thumbnail != ""):
      result = self.identityImage.thumbnail
    else:
      result = self.identicon
```